src/bridge/keybindings.rs

```diff
@@ -46,9 +46,9 @@ pub fn parse_keycode(keycode: Keycode) -> Option<(&'static str, bool)> {
         Keycode::Num9 => Some(("9", false)),
         Keycode::Colon => Some((":", false)),
         Keycode::Semicolon => Some((";", false)),
-        Keycode::Less => Some(("lt", true)),
```

Comment:
Special in this case means that the resulting keybinding should be wrapped in brackets. This is necessary because < and > are the escaping characters themselves. So we turn Keycode::Less into <lt> instead of the < character directly. I suspect that this change would mess up being able to type those characters at all. What made you make this change, and what happens if it is not present?

Reply:
I see! The issue that I'm having is that < and > are on the same button. So to get > I press shift + <. When `lt` returns false I can get > by pressing that key combination. `gt` being false I don't think has an effect but is there from when I was testing.

Reply:
Hrmmm... Your change would cause problems if someone binds < or > with a modifier. In your branch does holding ctrl and pressing < do something weird?

Reply:
If `lt` is true then I will get < even if I press shift + <. I'm guessing anything that will make this key combination return would work.

Reply:
I can't see any weird behaviour. But I'm not used to Vim. I'm only trying it out plainly in normal and insert mode.

Reply:
I get the same behaviour as when I run nvim in my Powershell terminal, that it tabs the row left and right one step if I do << and >> respectively.

Reply:
Interesting. In that case I'll pull and test it out locally

Reply:
Oh, sorry. I guess the weird behaviour is that I'm _able_ to output these characters. When I use nvim in my terminal I get the characters ^\ as output.

Reply:
What I'm confused then. Which is the desired behavior?

Reply:
I don't know for Ctrl + <. The only issue is that on a Nordic(?) keyboard you need to press Shift + < to output >.

Reply:
I guess you can do an extra case for (shift || !special) but that will affect a lot of characters and I don't know if that'd cause more trouble? The least intrusive but oddly specific I guess would be (key_text == "lt" && shift || !special). Apparently the behaviour I observed in terminal nvim for Ctrl + < and Ctrl + > doesn't exist on master for NeoVide either. So not exclusive for this request.

Reply:
its actually not too odd to special case < and > because they are the only keys which produce a character and also need to be escaped

So I actually think thats the best option.

Reply:
Thank you!

Reply:
No thank you! This has been bugging me for a while

Reply:
I will do some extra testing on my Linux machine on Monday, but at least I can use it on my Windows machine now! 👍 Been great using it so far!